src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -326,7 +320,7 @@ simulation = Simulation(model, Δt=1.25, stop_iteration=3)
 
 f(model) = model.clock.time^2; # scalar output
 
-g(model) = model.clock.time .* exp.(znodes(grid, Center())) # vector/profile output
+g(model) = model.clock.time .* exp.(znodes(Center, grid)) # vector/profile output
```

Comment:
You may want to merge `main` because I think we need this change for the doctest to pass

Reply:
I'm two commits ahead of main, none behind https://github.com/CliMA/Oceananigans.jl/compare/main...mk/compression I haven't actively changed these, but maybe @simone-silvestri and I started off from an outdated branch?

Reply:
yes not sure, but this change does walk back a recent PR

Reply:
shoot then I might have created a non-consistent history, sorry, I'll try to resolve that.